src/routes/Birthday.ts

```diff
@@ -0,0 +1,168 @@
+  }
+});
+
+/* router.use('get', async ({ msg, guildUser, em }) => {
+  // Toevoeging van een role op basis van ID
+  if (msg.member) {
+    const guild = await guildUser?.guild.birthdayRole;
+
+    if (guild !== undefined) {
+      const role = msg.member.roles.add(guild);
+      const sendMessage = msg.channel.send('You have gained a birthday.');
+
+      const today = moment().locale('nl').format('DD MMMM');
+      const discUser = msg.client.users.fetch(msg.member.id, true);
+
+      await role;
+      await sendMessage;
+    } else {
+      msg.channel.send('Er is nog geen role geselecteerd voor deze server');
+    }
+  }
+});
+*/
+
+router.use('help', async ({ msg }) => {
+  let message = '**Krijg elke ochtend een melding als iemand jarig is**\nHier volgen alle commando\'s voor verjaardagen';
+  message += '\n`ei bday show-all`: Laat alle geregistreerde verjaardagen zien';
+  message += '\n`ei bday set <DD/MM/YYYY>`: Registreerd jouw verjaardag';
+  message += '\n`ei bday delete`: Verwijderd jouw verjaardag';
+  message += '\n`ei bday check`: Laat zien wie er vandaag jarig is';
+  message += '\n***Admin Commando\'s***';
+  message += '\n`ei bday set-channel`: Selecteerd het huidige kanaal voor de dagelijkse update';
+
+  msg.channel.send(message);
+});
+
+router.use('show-all', async ({ msg, em }) => {
+  let message = 'Hier zijn alle verjaardagen die zijn geregistreerd: ';
+  const users = await em.find(User, { $not: { birthday: null } });
+  const discUsers = await Promise.all(users.map((u) => msg.client.users.fetch(u.id, true)));
+  discUsers.forEach((du) => { message += `\n${du.username} is geboren op ${moment(users.find((u) => u.id === du.id)?.birthday).locale('nl').format('DD MMMM YYYY')}`; });
+
+  msg.channel.send(message);
+});
+
+router.use('delete', async ({ msg, user }) => {
+  user.birthday = undefined;
+  msg.channel.send(`@${msg.author.tag}, je verjaardag is verwijderd.`);
+});
+
+router.use('check', async ({ em, msg }) => {
+  const today = moment().startOf('day').locale('nl').format('DD MMMM');
+  const todayAge = moment().startOf('day').locale('nl').format('YYYY');
+
+  const users = await em.find(User, { $not: { birthday: null } });
+  const discUsers = await Promise.all(users.map((u) => msg.client.users.fetch(u.id, true)));
+
+  let bdayToday = false;
+
+  let message = '**Deze mensen zijn vandaag jarig**';
+
+  discUsers.forEach((du) => {
+    const discBday = moment(users.find((u) => u.id === du.id)?.birthday).locale('nl').format('DD MMMM');
+    const discBdayAge = moment(users.find((u) => u.id === du.id)?.birthday).locale('nl').format('YYYY');
+    if (today === discBday) {
+      bdayToday = true;
+      const age = parseInt(todayAge, 10) - parseInt(discBdayAge, 10);
+      message += (`\n${du.username} is vandaag ${age} geworden!`);
+    }
+    if (bdayToday) {
+      msg.channel.send(message);
+    }
+  });
+});
```

Comment:
Moest deze functie niet weg?

Reply:
Ik dacht misschien leuk om er in te houden, maar hij kan in principe weg ja